sync3/handler/connstate.go

```diff
@@ -93,32 +93,40 @@ func (s *ConnState) load(ctx context.Context) error {
 	for _, metadata := range joinedRooms {
 		metadata.RemoveHero(s.userID)
 		urd := s.userCache.LoadRoomData(metadata.RoomID)
-		rooms[i] = sync3.RoomConnMetadata{
-			RoomMetadata: *metadata,
-			UserRoomData: urd,
+		timestamps := make(map[string]uint64, len(req.Lists))
```

Comment:
I tried using `s.muxedReq` here so that I wouldn't have to pass an argument through, but I got segfaults for my trouble. I guess this gets called before s.muxedReq is assigned a nonnil pointer.

Reply:
This needs a better name. I don't know what the keys are.

Reply:
d8f158b has a better name.